README.md

```diff
@@ -90,7 +90,7 @@ After you're finished please remove all the comments and instructions!
   <ul>
     <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
   </ul>
-</details>
+</details> -->
```

Comment:
- [ ] As stated by the previous reviewer, kindly add a tech stack for your project. for example, the client includes (HTML, CSS, and JS) and the database includes (localstorage).